### DISCUSSION TITLE
I added support for Typst in BeauTyXT, my text and markdown editor for Android!

### DISCUSSION THREAD

**Discussion by soupslurpr** [typst/typst#2625] [Show and tell]
I implemented Typst support in my app. Typst seems so nice, I can't wait to use it more. BeauTyXT is at https://github.com/soupslurpr/BeauTyXT I'm actually using it on laptop using Windows subsystem for Android 😎

**Comment by CreadorSW** [comment]
Hi! How can I get Typst in Android in order for BeauTyxT to show the preview?

**Comment by AsmanovLev** [comment]
For the newcomers of this topic:
>**Changelog v67**
>...
> • remove Typst support
> • update dependencies

Dropped because of [inability to upgrade](https://github.com/soupslurpr/BeauTyXT/issues/186)
